### PR TITLE
test: unit test for user util - getUsernamesFromPRs

### PR DESCRIPTION
**Pull request by khalatevarun**
Fixes - #1025 

Add unit test for user utils - `getUsernamesFromPRs()`

### CODE REVIEW

File: test/unit/utils/users.test.js [hunk 1]
```diff
@@ -65,4 +65,145 @@ describe("users", function () {
...
+        {
+          title: "Added a link on logo ",
+          username: "kushvahasumit",
+          state: "closed",
+          createdAt: "2023-04-29T18:01:24Z",
+          updatedAt: "2023-05-28T17:58:53Z",
+          repository: "website-www",
+          url: "https://github.com/Real-Dev-Squad/website-www/pull/465",
+          labels: [],
+          assignees: [],
```
Comment (prakashchoudhary07) [inline test/unit/utils/users.test.js:93]: What do you think about moving the mock data into fixtures?

File: test/unit/utils/users.test.js [hunk 2]
```diff
@@ -65,4 +66,31 @@ describe("users", function () {
...
     });
   });
+
+  describe("getUsernamesFromPRs", function () {
+    it("returns an array of unique usernames from the filtered PRs/Issues response", function () {
+      // Mock the allPRs data
+      const allPRs = filteredPRs;
+
+      // Execute the function and store the result
+      const usernames = usersUtils.getUsernamesFromPRs(allPRs);
```
Comment (shubhamsigdar1) [inline test/unit/utils/users.test.js:76]: getUsernamesFromPrs is a function that return all the usernames in an array?? Correct me if I am wrong??

**Review by shubhamsigdar1 — COMMENTED** [review]
great work @khalatevarun

File: test/unit/utils/users.test.js (hunk 2 above)
Comment (khalatevarun) [reply test/unit/utils/users.test.js:76]: yes

**Pull request merged**